### ISSUE TITLE
Script syntax error

### ISSUE THREAD

**Issue by yosefahab** [edgeimpulse/ei-install-scripts#2]
Hello,
You're missing semicolons ;; at the ends of lines 6 & 7 that are preventing the script from running.
```
while :; do
	case $1 in
		-l|--install-ei-linux) LINUX="true"
		-u|--update-all) UPDATE="true"
		;;
		*) break
	esac
	shift
done
```
Thanks!

**Comment by jonathanrjpereira** [comment]
@yosefahab So do you just have to make the following changes?

```
-l|--install-ei-linux) LINUX="true";;
-u|--update-all) UPDATE="true";;
```

**Comment by yennster** [comment]
@jonathanrjpereira Thanks for reporting! I will fix.

**Comment by Dasch0** [comment]
should be fixed in c56fbedc6c3d2c8ee03eeb62ff8af0f4d9a77473
